eureka_ml_insights/prompt_templates/aime_templates/Template_tag1.jinja

```diff
@@ -0,0 +1,19 @@
+You are a genius math expert in understanding math questions. Please read the following math question, and then decide which math category it falls into.
+
+Your judgment should be one of the following:
+
+arithmetic
+algebra
+counting
+geometry
+number theory
+probability
+other topics
+
+Do not generate any other texts except one of the above topics.
+
+----------
+Original question:
+{{prompt}}
+----------
+Your judgment:
```

Comment:
Do we want the model to only output a single tag per question or a list of tags? Or do we think the questions are usually strictly on one topic? Also, I am still not sure about the arithmetic tag if there would be any ambiguities here given that a good part of the questions require some arithmetic.

Reply:
changed the prompt to generate multiple topics